.gitignore

```diff
@@ -0,0 +1,20 @@
+# Compiled files
+__pycache__
+.pytest_cache
+.coverage
+
+# Packages/installer info
+*.egg
+*.eggs
+*.egg-info
+dist
+build
+_version.py
+
+# Mac OSX
```

Comment:
remove macos and pycharm specific things

Reply:
I am not sure I agree here.
E.g. having `.DS_Store` in the `.gitignore` helps all users on macos while it does not hurt anyone else.
Once added there, the maintainenance cost is null,  even if `.DS_Store` would disappear from macos in the future...
Even if there is no claim to maintain a gitignore for every OS or editor in use, I can see only benefits to covering a large portion of users.